test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -4896,6 +4899,69 @@ TEST_F(AtenXlaTensorTest, TestExpandSymIntStatic) {
   ExpectCounterChanged("xla::expand_symint", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestExpandSymIntSymbolic) {
+  torch::Tensor a = torch::ones({3, 4}, torch::TensorOptions(torch::kFloat));
+  torch::Tensor b = a.expand({2, 3, 4}, /*implicit=*/false);
+
+  // create a symbolic symint with value 2 and upperbound 2. This symint is
+  // symbolic(since it wraps around a sizeNode) but not really dynamic.
+  torch::lazy::Value scalar_value =
+      torch::lazy::Value(ScalarOp(1.0, xla::F32), 0);
+  std::vector<int64_t> target_size = {2, 3, 5};
```

Comment:
why is the last array element `5` and not `4`?

Reply:
I think it is a typo.. I will fix it